packages/app-metadata-controller/CHANGELOG.md

```diff
@@ -8,3 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 [Unreleased]: https://github.com/MetaMask/core/
```

Comment:
This line needs to be placed at the bottom of the file, then the changelog will pass validation.